modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryVersionLocalServiceTest.java

```diff
@@ -162,11 +162,12 @@ public void testAddObjectEntryVersion() throws Exception {
 				objectEntry.getObjectEntryId()));
 
 		objectEntry = _objectEntryLocalService.updateObjectEntry(
-			TestPropsValues.getUserId(), objectEntry.getObjectEntryId(),
+			objectEntry.getObjectEntryId(), TestPropsValues.getUserId(),
+			ObjectEntryFolderConstants.PARENT_OBJECT_ENTRY_FOLDER_ID_DEFAULT,
+			ServiceContextTestUtil.getServiceContext(),
```

Comment:
You can use `objectEntry.getObjectEntryFolderId()` here. Please, review all usages from this class.